service/auth.go

```diff
@@ -34,7 +35,7 @@ func checkPermissionsAndGetUserFromJWT(w http.ResponseWriter, r *http.Request, m
 }
 
 func getUserFromJWT(w http.ResponseWriter, r *http.Request) (datastore.User, error) {
-	token, err := JWTCheck(w, r)
+	token, err := utils.JWTCheck(w, r)
```

Comment:
Go doesn't recommend having a utils package. What about something like 'auth', 'sso' or even 'jwt'?. Actually, I think Go talibans would be happy with something like jwt.Check(w,r)